cmd/decode-mount-opts/decode-mount-opts.c

```diff
@@ -32,6 +32,7 @@ int main(int argc, char *argv[])
 		fprintf(stderr, "cannot parse given argument as a number\n");
 		return 1;
 	}
-	printf("%#lx is %s\n", mountflags, sc_mount_opt2str(mountflags));
+	char buf[1000];
+	printf("%#lx is %s\n", mountflags, sc_mount_opt2str(buf, sizeof buf, mountflags));
```

Comment:
(nitpick) I noticed we have this a lot in the code but I personally prefer the parenthese around `sizeof(buf)`. I know its not strictly needed but sometimes it and sometimes it is not and thats why I prefer consistency (but not a blocker if everyone else is fine with this).